src/components/pages/devices/devices.scss

```diff
@@ -4,4 +4,6 @@
 
 .devices-container {
   padding: 20px;
+
+  .status { color: #fff;}
```

Comment:
Should we be doing theming along the way?

Reply:
On the list for another PR.